lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -866,6 +896,31 @@ def test_pagination(self):
             }
         )
 
+    @ddt.data(
+        (True, "endorsed_comment"),
+        ("true", "endorsed_comment"),
+        ("1", "endorsed_comment"),
+        (False, "non_endorsed_comment"),
+        ("false", "non_endorsed_comment"),
+        ("0", "non_endorsed_comment"),
+    )
```

Comment:
here it is https://github.com/edx/edx-platform/pull/10736/files#diff-405a002934ecf4b172f4b87e5845b906R436